libraries/abstractions/common_io/include/iot_i2c.h

```diff
@@ -118,7 +117,7 @@ struct                    IotI2CDescriptor;
  *        This is initialized in open and returned to caller. The caller must pass
  *        this pointer to the rest of APIs.
  */
-typedef struct IotI2CDescriptor   * IotI2CHandle_t;
+typedef struct IotI2CDescriptor * IotI2CHandle_t;
```

Comment:
Where is IotI2CDescriptor defined?  Where do we tell implementors what values it must/should have?

Reply:
This anonymous struct is defined in iot_i2c.c, so board specific. Are you saying we should guide vendors how to implement iot_i2c.c?

Reply:
No, but I think we should guide them on what should go in the IotI2CDescriptor.  We can clarify that this is vendor/driver/component specific, but that it should probably include ...

Reply:
ok, I see. This information is only useful to vendor, not users. So I will put it to porting guide.